src/test/suite/integrationExplorer.test.ts

```diff
@@ -41,6 +41,7 @@ suite('Camel-k Integrations View', () => {
 
 	teardown(() => {
 		sandbox.restore();
+		integrationExplorer.setRetrieveIntegrations(true);
```

Comment:
important to avoid side-effect of having Integration explorer effectively showing integrations in the explorer

Reply:
Good catch!